File: HELLO_ANDROID01/app/src/main/java/kr/co/aiai/app/MainActivity04.java
package kr.co.aiai.app;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity04 extends AppCompatActivity {
    Button btn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main04);

        btn = findViewById(R.id.btn);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                myclick();
            }
        });
    }
    void myclick(){
        int[] arr=new int[45];

        for(int i=0;i<arr.length;i++){
            arr[i]=i;
        }
        for(int i=0;i<100000;i++){
            int rnd=(int)((Math.random()*arr.length-1)+1);
            int temp=arr[0];
            arr[0]=arr[rnd];
            arr[rnd]=temp;
        }

        TextView[] tvarr={findViewById(R.id.tv1),findViewById(R.id.tv2),findViewById(R.id.tv3),findViewById(R.id.tv4),
                findViewById(R.id.tv5),findViewById(R.id.tv6)
        };
        for(int i=0;i<tvarr.length;i++) {
            for(int j=0;j<tvarr.length;j++) {
                int a=arr[i];
                int b= arr[j];
                if(a<b) {
                    arr[i]=b;
                    arr[j]=a;
                }
            }
        }


        for(int i=0;i<tvarr.length;i++){
            tvarr[i].setText(""+arr[i]);
        }

    }
}

File: HELLO_AI4/ANDROID/app/src/main/java/kr/co/aiai/myapp/MainActivity04.kt
package kr.co.aiai.myapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import kotlin.random.Random

class MainActivity04 : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main04)
        var btn = findViewById<Button>(R.id.btn)

        btn.setOnClickListener{
            myclick()
        }


    }
    fun myclick(){
        var arr=IntArray(45)
        for (i in 0 until arr.size) {
            arr[i] = i + 1
        }
        for(i in 0 until 10000){
            val rnd = Random.nextInt(0, arr.size)
            var temp = arr[0]
            arr[0]=arr[rnd]
            arr[rnd]=temp
        }

        var tvarr= arrayOf(
            findViewById<TextView>(R.id.tv1)
            ,findViewById<TextView>(R.id.tv2)
            ,findViewById<TextView>(R.id.tv3)
            ,findViewById<TextView>(R.id.tv4)
            ,findViewById<TextView>(R.id.tv5)
            ,findViewById<TextView>(R.id.tv6)
        )

        for(i in 0 until tvarr.size){
            for(j in 0 until tvarr.size){
                var a=arr[i]
                var b=arr[j]
                if(a<b){
                    arr[i]=b
                    arr[j]=a;
                }
            }
        }

        for(i in 0 until tvarr.size){
            tvarr[i].setText(arr[i].toString())
        }
    }

}